app/javascript/components/Button.tsx

```diff
@@ -7,104 +7,152 @@ import { classNames } from "$app/utils/classNames";
 
 import { ButtonColor, buttonColors } from "$app/components/design";
 
-export const buttonVariants = cva("button", {
-  variants: {
-    variant: {
-      default: "",
-      outline: "",
-      secondary: "",
-      destructive: "",
-    },
-    size: {
-      default: "",
-      sm: "small",
-    },
-    color: {
-      primary: "primary",
-      black: "black",
-      accent: "accent",
-      filled: "filled",
-      success: "success",
-      danger: "danger",
-      warning: "warning",
-      info: "info",
-    },
-  },
-  compoundVariants: [
-    {
-      variant: "outline",
-      color: "primary",
-      className: "outline-primary",
-    },
-    {
-      variant: "outline",
-      color: "danger",
-      className: "outline-danger",
-    },
-    {
-      variant: "outline",
-      color: "success",
-      className: "outline-success",
-    },
-    {
-      variant: "outline",
-      color: "warning",
-      className: "outline-warning",
-    },
-    {
-      variant: "outline",
-      color: "info",
-      className: "outline-info",
-    },
-    {
-      variant: "outline",
-      color: "black",
-      className: "outline-black",
-    },
-    {
-      variant: "outline",
-      color: "accent",
-      className: "outline-accent",
+export type BrandName = "paypal" | "discord" | "stripe" | "facebook" | "twitter" | "apple" | "android" | "kindle" | "zoom" | "google";
+
+const brandColors: Record<BrandName, { bg: string; text: string }> = {
+  paypal: { bg: "#00457c", text: "#ffffff" },
+  discord: { bg: "#7289da", text: "#ffffff" },
+  stripe: { bg: "#625bf6", text: "#ffffff" },
+  facebook: { bg: "#4267b2", text: "#ffffff" },
+  twitter: { bg: "#000000", text: "#ffffff" },
+  apple: { bg: "#000000", text: "#ffffff" },
+  android: { bg: "#142f40", text: "#ffffff" },
+  kindle: { bg: "#f3a642", text: "#000000" },
+  zoom: { bg: "#4087fc", text: "#ffffff" },
+  google: { bg: "#5383ec", text: "#ffffff" },
+};
+
+export const buttonVariants = cva(
+  "inline-flex items-center justify-center gap-2 cursor-pointer border border-border rounded bg-transparent font-inherit no-underline transition-transform hover:-translate-x-1 hover:-translate-y-1 hover:shadow-[0.25rem_0.25rem_0_currentColor] active:translate-x-0 active:translate-y-0 active:shadow-none disabled:opacity-30 disabled:hover:translate-x-0 disabled:hover:translate-y-0 disabled:hover:shadow-none",
+  {
+    variants: {
+      variant: {
+        default: "",
+        outline: "bg-transparent",
+        secondary: "",
+        destructive: "",
+      },
+      size: {
+        default: "px-4 py-3 text-base leading-[1.4]",
+        sm: "p-2 text-sm leading-[1.3]",
+      },
+      color: {
+        primary: "bg-black text-white hover:bg-[rgb(var(--accent))] hover:text-[rgb(var(--contrast-accent))]",
```

Comment:
We shouldn't be using variables directly in Tailwind - please use the colours in the Tailwind config (e.g. `bg-accent`) and add any that are missing